Cartesian_kernel/include/CGAL/Cartesian/Circle_3.h

```diff
@@ -130,7 +130,7 @@ class CircleC3 {
     return diametral_sphere();
   }
 
-  Point_3 center() const
+  const Point_3& center() const
```

Comment:
```suggestion
  declspec(auto) center() const
```

There is no guarantee that the rep `Sphere_3` is the one we know. Let's keep this code generic.